### PR TITLE
Freeze value objects after construction

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: src/Validation/GetInTouchValidator.php [hunk 1]
```diff
@@ -22,16 +22,17 @@ public function __construct( MailValidator $mailValidator ) {
...
 
 	/**
-	 * @param GetInTouchRequest $instance
-	 * @return bool
+	 * @param GetInTouchRequest $donation
+	 *
+*@return bool
 	 */
-	public function validate( $instance ): bool {
+	public function validate( $donation ): bool {
```
Comment (KaiNissen) [inline src/Validation/GetInTouchValidator.php]: This and the instances below are not donations.

**Comment by KaiNissen** [comment]
Renaming spree. :)

**Comment by JeroenDeDauw** [comment]
#FailStorm

**Comment by JeroenDeDauw** [comment]
Fixed rename fail and rebased

**Pull request merged**